java/test/jmri/implementation/AbstractStringIOTest.java

```diff
@@ -27,6 +27,15 @@ public void testSystemNames() {
         Assert.assertEquals("StringIO system name is correct", "IZ01", myStringIO_2.getSystemName());
         Assert.assertNotEquals("StringIOs are different", myStringIO_1, myStringIO_2);
         Assert.assertNotEquals("StringIO compareTo returns not zero", 0, myStringIO_1.compareTo(myStringIO_2));
+        
+        Assert.assertEquals("compareSystemNameSuffix returns correct value",
```

Comment:
Since this is testing a method of the class, please make it a separate test, rather than lumping it in with another test.